README.md

```diff
@@ -1,4 +1,50 @@
-# administrivia
-repository to track meta-work.
+# Administrivia
+
+This repository tracks PING's meta-work.
 
 2019 charter draft: https://w3cping.github.io/administrivia/charter-draft.html
+
+# PING Process
+
+PING's goal is to ensure that the specifications recommended by the W3C respect the privacy of people who use the web platform, and ensure that the web platform is an environment which allows people to browse the web privately.
+
+PING's core administrative mechanism for achieving this is _horizontal review_: members of PING review specifications and provide recommendations to the authors about how to ensure that their specifications support a web platform which respects privacy. This formal review process is the most visible part of a larger iceberg of activities aimed at helping specification authors write with privacy in mind. The best place to make privacy-related improvements to specifications is substantially in advance of horizontal review points. PING welcomes conversations with authors at all phases of specification development. Ideally, PING would be thoroughly familiar with specifications, their potential privacy impacts, and the measures takes to mitigate those impacts well before the time horizontal review points are reached, and those privacy reviews would consist mostly of codifying & recording conversations which have already happened, rather than pointing new things out to authors.
```

Comment:
PING has a privacy questionnaire at https://w3c.github.io/ping/privacy-questions.html. Is that expected to flag enough issues that specification authors can reliably write with privacy in mind, or do you expect every specification to need a conversation with PING members during the early design phase in order to be generally on the right track?